controllers/usersController.js

```diff
@@ -61,15 +46,13 @@ UsersController.delete = function(req, res){
 
```

Comment:
My bad, I'll go back to doing the same function headers I used to do. Got a bit lazy 😅 When you say the why, could you elaborate? Are you talking more like "explain why this line/block of code is needed" or are you talking more on a function scale?

Reply:
My bad, looking at this I realize that I thought this method was more confusing because I had mixed it up with the following method when looking at the changes.

My point still stands though, if you had to try more than once to write a function, or if it has some logic that was influenced by other parts of the code, do your best to explain _why_ you made wrote the code the way you did, rather than just explaining _what_ the code does.